src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -147,16 +153,40 @@ def ensure_output_dirs_exist(self):
 class RscCompile(ZincCompile):
   """Compile Scala and Java code to classfiles using Rsc."""
 
-  _name = 'rsc' # noqa
+  _compiler_lookup = {
+    # NB It is set up this way to simplify the tag names, but it would be more precise to have the
+    # tags use the workflow names directly
+    'rsc': 'rsc-then-zinc',
```

Comment:
I would prefer using the workflow names directly (and using `_JvmCompileWorkflow` instances instead of strings).

Reply:
that works for me.